addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -328,6 +328,125 @@ options.registry.sizing_x = options.registry.sizing.extend({
+    },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * Add new column which is clone of last column
+     *
+     * @private
+     */
+    _addColumn: function () {
+        this.trigger_up('request_history_undo_record', {$target: this.$target});
+        var $lastColumn = this._getColumns().last();
+        $lastColumn.clone().insertAfter($lastColumn);
+        this._updateSnippet();
+    },
+    /**
+     * Return all the columns of currently selected snippet.
+     *
+     * @private
+     * @returns {jQuery}
+     */
+    _getColumns: function () {
+        return this.$target.find('.container > .row > [class*="col-md-"]');
+    },
+    /**
+     * Remove last column
+     *
+     * @private
+     */
+    _removeColumn: function () {
+        this.trigger_up('request_history_undo_record', {$target: this.$target});
+        var $lastColumn = this._getColumns().last();
+        $lastColumn.remove();
+        this._updateSnippet();
+    },
+    /**
+     * Resize the columns
+     *
+     * @private
+     */
+    _resizeColumns: function () {
+        var $columns = this._getColumns();
+        var colsLength = $columns.length;
+        if (colsLength <= 6) {
+            var colSize = Math.round(12 / colsLength);
+            $columns.each(function () {
+                $(this).attr('class', $(this).attr('class').replace(/\s*(col-md-|col-md-offset-)([0-9-]+)/g, ''));
+                $(this).addClass('col-md-' + colSize);
+            });
+            if (colsLength == 5) {
+                $columns.first().addClass('col-md-offset-1');
+            }
+        }
+    },
+    /**
+     * Toggle plus/minus button
+     *
+     * @private
+     */
+    _toggleButtons: function () {
+        var colsLength = this._getColumns().length;
+        var $addButton = this.$overlay.find('.o_handle.o_add_column');
+        var $removeButton = this.$overlay.find('.o_handle.o_remove_column');
+        if (colsLength >= 6) {
+            $addButton.addClass('o_disable_btn');
+            $removeButton.removeClass('o_disable_btn');
+        } else if (colsLength > 1) {
+            $addButton.removeClass('o_disable_btn');
+            $removeButton.removeClass('o_disable_btn');
+        } else {
+            $addButton.removeClass('o_disable_btn');
+            $removeButton.addClass('o_disable_btn');
+        }
+    },
+    /**
+     * @private
+     */
+    _updateSnippet: function () {
+        this._resizeColumns();
+        this._toggleButtons();
+        this.trigger_up('cover_update');
+    },
+
+    //--------------------------------------------------------------------------
+    // Handlers
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     * @param {MouseEvent} event
+     */
+    _onButtonClick: function (event) {
```

Comment:
Name your event variable `ev` instead of `event`. Chrome defines a magic `event` variable, so if someone use `event` without declaring it, this may causes error on other browsers (so difficult to see).